test/browser/column/resizable.test.tsx

```diff
@@ -66,48 +66,52 @@ test('cannot not resize or auto resize column when resizable is not specified',
 test('should resize column when dragging the handle', async () => {
   const onColumnResize = vi.fn();
   setup<Row, unknown>({ columns, rows: [], onColumnResize });
-  const [, col2] = getHeaderCells();
   const grid = getGrid();
   expect(onColumnResize).not.toHaveBeenCalled();
   await expect.element(grid).toHaveStyle({ gridTemplateColumns: '100px 200px' });
+  const [, col2] = getHeaderCells();
```

Comment:
I don't understand, surely the cell element should still be in the document if we get it earlier, no?

Reply:
It should be. I will investigate in a separate PR